docs/csharp/whats-new/tutorials/snippets/staticinterfaces/staticinterfaces.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
```suggestion
    <OutputType>Exe</OutputType>
    <LangVersion>Preview</LangVersion>
```

Reply:
Adding the `Preview` element for `LangVersion` isn't necessary. Once the `EnablePreviewFeatures` is set to `true`, the language version tracks to `Preview`.